src/platform/forms-system/src/js/validation.js

```diff
@@ -376,7 +384,7 @@ export function validateCurrentOrPastDate(
   const {
     futureDate = 'Please provide a valid current or past date',
   } = errorMessages;
-  validateDate(errors, dateString);
+  validateDate(errors, dateString, minYear, moment().year());
```

Comment:
Is it possible to just use the [JavaScript date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) here instead of moment? Moment isn't being maintained anymore, so it's probably good to not introduce it new places.

Reply:
Sure, good idea! My change is pushed.